src/main/java/it/gov/pagopa/ranking/service/RankingRequestsApiServiceImpl.java

```diff
@@ -0,0 +1,83 @@
+package it.gov.pagopa.ranking.service;
+
+import it.gov.pagopa.ranking.dto.RankingRequestsApiDTO;
+import it.gov.pagopa.ranking.dto.mapper.OnboardingRankingRequest2RankingRequestsApiDTOMapper;
+import it.gov.pagopa.ranking.model.InitiativeConfig;
+import it.gov.pagopa.ranking.model.OnboardingRankingRequests;
+import it.gov.pagopa.ranking.model.Order;
+import it.gov.pagopa.ranking.repository.OnboardingRankingRequestsRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import static it.gov.pagopa.ranking.utils.RankingConstants.CRITERIA_CODE_ISEE;
+
+@Service
+@Slf4j
+public class RankingRequestsApiServiceImpl implements RankingRequestsApiService {
+
+    private final OnboardingRankingRequestsRepository onboardingRankingRequestsRepository;
+    private final OnboardingRankingRequest2RankingRequestsApiDTOMapper dtoMapper;
+    private final RankingContextHolderService rankingContextHolderService;
+
+    public RankingRequestsApiServiceImpl(OnboardingRankingRequestsRepository onboardingRankingRequestsRepository, OnboardingRankingRequest2RankingRequestsApiDTOMapper rankingRequestsApiDTOMapper, RankingContextHolderService rankingContextHolderService) {
+        this.onboardingRankingRequestsRepository = onboardingRankingRequestsRepository;
+        this.dtoMapper = rankingRequestsApiDTOMapper;
+        this.rankingContextHolderService = rankingContextHolderService;
+    }
+
+    @Override
+    public List<RankingRequestsApiDTO>  findByInitiativeId(String organizationId, String initiativeId, int page, int size) {
+        LocalDate today = LocalDate.now();
+
+        InitiativeConfig initiative = rankingContextHolderService.getInitiativeConfig(initiativeId, organizationId);
+        if (initiative == null) {
+            return null;
+        } else {
+            List<RankingRequestsApiDTO> out = new ArrayList<>();
+
+            if (!initiative.getRankingEndDate().isBefore(today)) {
```

Comment:
The endDate is still opened to new onboarding! Thus it should not return any data
```suggestion
            if (initiative.getRankingEndDate().isAfter(today)) {
```